src/App.js

```diff
@@ -3,11 +3,13 @@ import { inflateSync } from 'browserify-zlib';
 import { makeStyles } from '@material-ui/core/styles';
 import { Grid } from '@material-ui/core';
 import { expandLink } from './utils/BitlyWorker';
+import { setExampleText } from './utils/ParseExamples';
 import TopBar from './components/TopBar';
 import JSONOutput from './components/JSONOutput';
 import ConsoleComponent from './components/ConsoleComponent';
 import CodeMirrorComponent from './components/CodeMirrorComponent';
 import SUSHIControls from './components/SUSHIControls';
+import config from './examples/examples-config.json';
```

Comment:
I added some documentation to the README, but I didn't add anything to the file itself since it is a JSON file and doesn't support comments. If you think it would be helpful to have the documentation closer to the actual file, I could convert it to just a `.js` file with an object or I could add a `examples-config.example.json`. Let me know what you think of those options and with the content of what I added!

Reply:
Nah, readme is fine, looks good!